src/main/kotlin/me/partlysanestudios/partlysaneskies/data/skyblockdata/Rarity.kt

```diff
@@ -0,0 +1,44 @@
+package me.partlysanestudios.partlysaneskies.data.skyblockdata
+
+/**
+ * An Enum class for all rarities in Hypixel Skyblock
+ * @param order: an integer signifying the rarity of the pet to be able to compare using operators
+ * @param colorCode: the color code relating to that rarity
+ * @param displayName: a prettier version of the enum name used for display
+ */
+enum class Rarity(val order: Int, val colorCode: String, val displayName: String): Comparable<Rarity> {
+    UNKNOWN(-1, "", "Unknown"),
+    COMMON(0, "§f", "Common"),
+    UNCOMMON(1, "§a", "Uncommon"),
+    RARE(2, "§9", "Rare"),
+    EPIC(3, "§5", "Epic"),
+    LEGENDARY(4, "§6", "Legendary"),
+    MYTHIC(5, "§d", "Mythic"),
+    DIVINE(6, "§b", "Divine"),
+    SUPREME(7, "§4", "Supreme"),
```

Comment:
Supreme got replaced with ULTIMATE_COSMETIC

Reply:
<img width="408" alt="image" src="https://github.com/PartlySaneStudios/partly-sane-skies/assets/83100266/c51b8f50-ea8d-4f87-9996-4c8ca1621bbe">
like this?

Reply:
Thats still supreme lol

Reply:
Whats the display name for it?

Reply:
Ultimate cosmetic - check those items in the  neu search list

Reply:
Done